tests/benchmarks/current_consumption/nfc_idle/sysbuild.cmake

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(SB_CONFIG_SOC_NRF54H20)
+  if("${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")
+    message(FATAL_ERROR "REMOTE_BOARD must be set to a valid board name")
+  endif()
```

Comment:
with the proposed changes in `Kconfig.sysbuild`, then as soon the nRF54h20 is used then the remote board is always set, so in principle this check can be removed.

But as it does no harm I leave the decision to the author.